core/src/data/tileData.h

```diff
@@ -66,73 +66,20 @@ typedef std::vector<Point> Line;
 
 typedef std::vector<Line> Polygon;
 
-int polygonSize(const Polygon& _polygon);
-
-struct Properties {
-
-    struct Item {
-        Item(std::string _key, Value _value) :
-            key(std::move(_key)), value(std::move(_value)) {}
-
-        std::string key;
-        Value value;
-        bool operator<(const Item& _rhs) const { return key < _rhs.key; }
-    };
-
-    Properties() {}
-    Properties(const Properties& _other) = default;
-    Properties(std::vector<Item>&& _items);
-    Properties& operator=(Properties&& _other);
-
-    const Value& get(const std::string& key) const;
-
-    void sort();
-
-    void clear() { props.clear(); }
-
-    bool contains(const std::string& key) const {
-        return !get(key).is<none_type>();
-    }
-
-    bool getNumeric(const std::string& key, float& value) const {
-        auto& it = get(key);
-        if (it.is<float>()) {
-            value = it.get<float>();
-            return true;
-        }
-        return false;
-    }
-
-    float getNumeric(const std::string& key) const {
-        auto& it = get(key);
-        if (it.is<float>()) {
-            return it.get<float>();
-        }
-        return 0;
-    }
-    bool getString(const std::string& key, std::string& value) const {
-        auto& it = get(key);
-        if (it.is<std::string>()) {
-            value = it.get<std::string>();
-            return true;
+struct Geometry {
```

Comment:
Why a struct here? We can have a Geometry namespace here, and make it an inline function.. (maybe ... ).